### DISCUSSION TITLE
Αριθμός ασκήσεων γραμμής εντολών

### DISCUSSION THREAD

**Discussion by p17balo** [courses-ionio/hci#1673] [Announcements]
ΚΑΛΗΣΠΕΡΑ , ΟΝΟΜΑΖΟΜΑΙ ΣΠΥΡΙΔΩΝ ΜΠΑΛΩΜΕΝΟΣ ΜΕ ΑΜ : Π2017032 , ΕΙΜΑΙ 5ο ΕΤΟΣ ΚΑΙ ΕΧΩ ΜΙΑ ΑΠΟΡΙΑ ΣΤΟΝ ΑΡΙΘΜΟ ΑΣΚΗΣΕΩΝ ΓΡΑΜΜΗΣ ΕΝΤΟΛΩΝ ΠΟΥ ΠΡΕΠΕΙ ΝΑ ΓΙΝΟΥΝ.  ΣΤΑ ΠΑΡΑΔΟΤΕΑ Ο ΑΡΙΘΜΟΣ ΕΙΝΑΙ 6 ΕΝΩ ΣΤΟΝ ΥΠΟΛΟΓΙΣΜΟ ΤΟΥ ΤΕΛΙΚΟΥ ΒΑΘΜΟΥ ΓΡΑΦΕΙ ΟΤΙ ΕΙΝΑΙ 2.

**Comment by AlexandrosV20** [comment]
Οι πρώτες εβδομάδες μέχρι την 6η είναι για την εγκατάσταση του λειτουργικού. Πρέπει να γράψεις για την διαδικασία της εγκατάστασης και τυχόν προβλήματα που παρουσιάστηκαν. Μετά τη 6η εβδομάδα πάνε οι ασκήσεις γραμμής εντολών σαν παραδοτέα.

**Comment by FaidonTsetsis** [comment]
Οι 6 πρώτες ασκήσεις αφορούν την εγκατάσταση του λειτουργικού συστήματος, που συνεπάγεται με μία στοιχειώδης περιγραφή της εμπειρίας σου με την εγκατάσταση, καθώς και πιθανών λαθών που προέκυψαν. Επίσης, μπορείς να παρουσιάσεις τα στοιχεία του λειτουργικού σου χρησιμοποιώντας κάποιες εντολές σαν τις neofetch ή την journalctl, και προαιρετικά να κάνεις καμία άσκηση από αυτές της κατηγορίας warm-up. Συνολικά από όλες τις εβδομάδες παραδοτέων για τις ασκήσεις γραμμής εντολών εσύ πρέπει να κάνεις το ελάχιστο δύο (αν κάνεις περισσότερες βαθμολογούνται οι δύο καλύτερές σου) από την κατηγορία hci.

**Comment by p14kairas** [comment]
@p17balo Με πιο απλά λόγια συνολικά θα χρειαστείς τουλάχιστον 2 ασκήσεις γραμμής εντολών για να βαθμολογηθείς πλήρως σε αυτό το κομμάτι. (Όπως αναφέρεται ήδη οι πρώτες 6 δεν χρειάζεται να γίνουν και αντί αυτών πρέπει να αναφέρεις την εγκατάσταση του λειτουργικού arch linux δηλ. όλη την διαδικασία που έκανες τι προβλήματα αντιμετώπισες παρατηρήσεις κλπ)
